httpbin/helpers.go

```diff
@@ -230,3 +232,26 @@ func sha1hash(input string) string {
 	h := sha1.New()
 	return fmt.Sprintf("%x", h.Sum([]byte(input)))
 }
+
+func isValidUUIDv4(uuid string) error {
```

Comment:
Let's put this function in the test file, to help make it obvious that it's only used as a test helper.

Reply:
Makes sense. Will do.

Reply:
Done.